Republik.ch.js

```diff
@@ -0,0 +1,388 @@
+{
+	"translatorID": "b4c83513-921f-474b-adc3-7017ea690ba7",
+	"label": "Republik.ch",
+	"creator": "Johannes Wüthrich",
+	"target": "^https?://www\\.republik\\.ch/[0-9]*/[0-9]*/[0-9]*/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
```

Comment:
This can be `gcsibv`! We're thankfully past the days of significant browser incompatibilities, at least when it comes to translators.

And re: being web-only, that's correct and shouldn't change. Import, search, etc., are for translators that allow you to import new metadata formats, locate items via ISBN, that kind of thing. Not relevant for a translator like this.

Reply:
Changed in 753c2ba